.github/workflows/ci.yml

```diff
@@ -456,6 +457,18 @@ jobs:
           PGPASSWORD=postgres psql -h localhost -p 5432 -U postgres -d postgres -c "ALTER SYSTEM SET wal_level = 'logical';"
           PGPASSWORD=postgres psql -h localhost -p 5432 -U postgres -d postgres -c "SELECT pg_reload_conf();"
 
+      - name: Restart PostgreSQL to apply wal_level
+        run: |
```

Comment:
oops, we need to set timeout to all operations; I met situations where minio / fake GCS setup gets stuck.